app/controllers/dish_controller.rb

```diff
@@ -20,6 +20,22 @@ def create
     end
   end
 
+  def edit
+   @menu = Menu.find(params[:menu_id])
```

Comment:
I have started with  ```@dish = Dish.find(params[:id])``` only but it gives an error of undefined method '`restaurant`. I think this makes sense because we are using nested resources.